### ISSUE TITLE
Option to export analysis results

### ISSUE THREAD

**Issue by kjappelbaum** [gadorlhiac/PyLDM#4]
What do you think about adding an option to export the LDM maps, e.g. using `np.savetxt()` to allow the users to replot the results of the analysis?

**Comment by gadorlhiac** [comment]
True, this is a good idea.  In your fork I notice you are saving each array to an individual file which may be tedious though, e.g. sifting through 100+ files would be time consuming.  A single text file, could work, given a format that specifies parameters at the beginning and groups the arrays by association, e.g. to an alpha value.  Alternatively, it could be written to an hdf5 file with every array being saved, and the hierarchy being maintained.  Do you think one of these ideas would work?

**Comment by kjappelbaum** [comment]
Indeed, the solution I implemented yesterday was rather quick and dirty as I needed the data and then simply used a regex to extract the alphas from the filenames,.
hdf5 (using h5py) grouped by e.g. alphas indeed might be a good way to go.

The ideal final solution (for me) would also save some context like the analysis parameters.
